### PR TITLE
translate what-is-dogecoin.md into Chinese

### PR DESCRIPTION
**Pull request by shuferhoo**
translate what-is-dogecoin.md into Chinese

### CODE REVIEW

**Comment by shuferhoo** [comment]
translate what-is-dogecoin.md into Chinese

**Review by gongwensi2020 — APPROVED** [review]
good translation!
好翻译

File: content/zh-cn/dogepedia/articles/what-is-dogecoin.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
 +++
-title = "What is Dogecoin?"
-date = "2021-10-22"
+title = "狗狗币是什么?"
+date = "2022-08-07"
 type = "article"
 
 [ author ]
-  name = "Dogecoin"
+  name = "Tiger Hu"
```
Comment (bobdos) [inline content/zh-cn/dogepedia/articles/what-is-dogecoin.md:7]: please keep the original author to 'Dogecoin' as it's an official written article. 

many thanks.

File: content/zh-cn/dogepedia/articles/what-is-dogecoin.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 +++
  
 ![image info](/assets/images/dogepedia/3.png)
-Dogecoin is many things to many different people. At its heart, Dogecoin is the accidental crypto **movement** that makes people smile! 
+狗狗币对不同人而言意味着不同的含义。 本质来讲，狗狗币是一场偶然发起的给人们带来快乐的加密货币运动！ 
 
-It is also an open-source peer-to-peer cryptocurrency that utilises [blockchain technology](https://www.dogecoin.com/dogepedia/articles/what-is-a-blockchain), a highly secure decentralised system of storing information as a public ledger that is maintained by a network of computers called [nodes](https://www.dogecoin.com/dogepedia/articles/what-is-a-node). 
+狗狗币是一种运用[区块链技术](https://www.dogecoin.com/dogepedia/articles/what-is-a-blockchain)的开源分布式数字货币 , 通过运用由计算机[节点](https://www.dogecoin.com/dogepedia/articles/what-is-a-node)组成的网络进行记账实现了一个高度安全的去中心化的系统。
 
-More than this, though, is the ethos of Dogecoin, summarised in the [Dogecoin Manifesto](https://foundation.dogecoin.com/manifesto/), and its amazing, vibrant community made up of friendly folks just like you!
```
Comment (bobdos) [inline content/zh-cn/dogepedia/articles/what-is-dogecoin.md:15]: url should be changed to:

* https://foundation.dogecoin.com/zh-cn/manifesto/

File: content/zh-cn/dogepedia/articles/what-is-dogecoin.md [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
  
 ![image info](/assets/images/dogepedia/3.png)
-Dogecoin is many things to many different people. At its heart, Dogecoin is the accidental crypto **movement** that makes people smile! 
+狗狗币对不同人而言意味着不同的含义。 本质来讲，狗狗币是一场偶然发起的给人们带来快乐的加密货币运动！ 
 
-It is also an open-source peer-to-peer cryptocurrency that utilises [blockchain technology](https://www.dogecoin.com/dogepedia/articles/what-is-a-blockchain), a highly secure decentralised system of storing information as a public ledger that is maintained by a network of computers called [nodes](https://www.dogecoin.com/dogepedia/articles/what-is-a-node). 
+狗狗币是一种运用[区块链技术](https://www.dogecoin.com/dogepedia/articles/what-is-a-blockchain)的开源分布式数字货币 , 通过运用由计算机[节点](https://www.dogecoin.com/dogepedia/articles/what-is-a-node)组成的网络进行记账实现了一个高度安全的去中心化的系统。
 
-More than this, though, is the ethos of Dogecoin, summarised in the [Dogecoin Manifesto](https://foundation.dogecoin.com/manifesto/), and its amazing, vibrant community made up of friendly folks just like you!  
+更重要的，狗狗币所蕴含的精神，如[狗狗币宣言](https://foundation.dogecoin.com/manifesto/)中所言，是一个由你我一样友善的人组成的令人惊叹、充满活力的社区！
```
Comment (bobdos) [inline content/zh-cn/dogepedia/articles/what-is-dogecoin.md:15]: url should be changed to:

https://foundation.dogecoin.com/zh-cn/manifesto/

**Review by bobdos — CHANGES_REQUESTED** [review]

File: content/zh-cn/dogepedia/articles/what-is-dogecoin.md (hunk 2 above)
Comment (shuferhoo) [reply content/zh-cn/dogepedia/articles/what-is-dogecoin.md:15]: ok

**Review by bobdos — APPROVED** [review]
all good and approved.

many thanks for your contribution and quick response.

cheers,

**Comment by shuferhoo** [comment]
> all good and approved.
> 
> many thanks for your contribution and quick response.
> 
> cheers,

my pleasure, but i can't see it on the main branch and website, why?

**Pull request merged**